models/users.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const users = (sequelize, DataTypes) => {
+  const users = sequelize.define('users', {
```

Comment:
change the model name, it has to be in the singular form.
```suggestion
  const users = sequelize.define('User', {
```